docs/docs/filtering/partial-evaluation.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Evaluating a data filter policy
+sidebar_position: 2
+---
+
+import SideBySideContainer from '@site/src/components/SideBySide/Container';
+import SideBySideColumn from '@site/src/components/SideBySide/Column';
+
+To understand how SQL WHERE clauses can be derived from a partially evaluated Rego policy, it's beneficial to have a basic idea about how _partial evaluation_ (PE) works.
+In this walk-through of a PE run, we'll start with a basic filter policy:
+
+```rego title="filters.rego"
+# METADATA
+# scope: package
+# compile:
+#   unknowns:
+#     - input.users
+#     - input.products
+package filters
+
+user := input.user
+
+include if {
+        input.users.name == user
+        input.budget != "low"
+}
+
+include if {
+        input.users.name == user
+        input.budget == "low"
+        input.products.price < 500
+}
+
+include if input.products.price == "free"
+```
+
+In this walkthrough, we'll go through the policy in the same way the evaluator does, and use the following input:
+
+```json
+{
+  "user": {
+    "name": "dana"
+  },
+  "budget": "low"
+}
+```
+
+---
+
+<SideBySideContainer>
+<SideBySideColumn>
+
+```rego
+user := input.user
+
+# highlight-next-line
+include if {
+        input.users.name == user
+        input.budget != "low"
+}
+
+include if {
+        input.users.name == user
+        input.budget == "low"
+        input.products.price < 500
+}
+
+include if input.products.price == "free"
+```
+</SideBySideColumn>
+<SideBySideColumn>
+The first `include` rule is evaluated.
+</SideBySideColumn>
+</SideBySideContainer>
+
+---
```

Comment:
Is there any reason in particular reason to have these? We tend not to have them quite so often on other pages.

Reply:
It's separating the steps... The content would work better as a slide-show, but that's too much for the docs. I can remove them, but I think it doesn't work out too bad this way, do you?

Reply:
Yeah I don't have strong feelings - perhaps numbered steps would make it clear too? just an idea...